packages/reassure-compare/src/utils/validate.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from 'zod';
+import { hasDuplicateValues } from './array';
+
+const metadataSchema = z.object({
```

Comment:
This entry should be split into two:
* `PerformanceMetadataSchema` + infer
* `PerformanceHeaderSchema` + infer